README.md

```diff
@@ -326,22 +326,18 @@ spec:
 Based on the Service Catalogues, Runtime Groups, Teams, and IdP Mappings we have created so far, we can assert that the following statements stand true for the ACME Konnect account:
 
 1. Users who are in the IdP Retail dev group of ACME Bank are able to create new services in the Retail service catalogue and these services will automatically be available to other members of their team. 
-
 2. Users who are in the IdP Investment dev group of ACME Bank are able to create new services in the Investment service catalogue and these services will automatically be available to other members of their team.
-
 3. Both Retail and Investment dev group members are able to publish their services to the ACME Common service catalogue so that they are available to any ACME Bank user who logs into ServiceHub and looks at the common catalogue.
-
 4. All ACME Bank employees are able to view all services in the Konnect ServiceHub Common catalogue. 
-
-4. Users who are in the IdP Retail dev group of ACME Bank are able to see and access the Retail Sandbox Runtime Group in Runtime Manager. They are thus able to:
-	5.	Start/stop Kong Gateway runtime instances that are associated with this runtime group.
-	6. Deploy their Services from ServiceHub to this runtime group 
-5. Users who are in the IdP Investment dev group of ACME Bank are able to see and access the Investment Sandbox Runtime Group in Runtime Manager. They are thus able to:
-	5.	Start/stop Kong Gateway runtime instances that are associated with this runtime group.
-	6. Deploy their Services from ServiceHub to this runtime group 
-6. Users who are in the ACME Operations IdP group are able to see and access the Production Runtime Group in Runtime Manager. They are thus able to: :
-	5.	Start/stop Kong Gateway runtime instances that are associated with this runtime group.
-	6. Deploy any Service to which they have Deploy access from ServiceHub to this runtime group
+5. Users who are in the IdP Retail dev group of ACME Bank are able to see and access the Retail Sandbox Runtime Group in Runtime Manager. They are thus able to:
+    1. Start/stop Kong Gateway runtime instances that are associated with this runtime group. [Rob] Start/stop a data plane seems like it may interfere with autoscaling groups or other infrastructure concerns in unpredictable ways. Since we are counting on customers to deploy their data planes in whatever method they are using, does this really mean add/remove a config from the Runtime group? This effectively stops the service from being reached through the gateway, and means that other groups that may be using the same data plane do not need to worry about some other group bringing down a gateway they depend on.
```

Comment:
Meaning that  they are able to use the provisioning ID for the runtime group to start/stop runtime instances in whchever form their want (Terraform, docker run, k8s autoscaling, ...). The runtime instances retrieve their config from the runtime group. Does that help?

Reply:
OK, I think I get it now. It means they have access to getting the token/provisioning id for that runtime group.

I will update the PR.